packages/editor-ui/vue.config.js

```diff
@@ -27,6 +28,12 @@ module.exports = {
 		plugins: [
 			new MonacoWebpackPlugin({ languages: ['javascript', 'json', 'typescript'] }),
 		],
+		resolve: {
+			alias: {
+				'element-ui/packages/button': path.resolve(__dirname, '..', 'design-system/src/components/N8nButton/Button.vue'),
+				'element-ui/lib/button': path.resolve(__dirname, '..', 'design-system/src/components/N8nButton/Button.vue'),
```

Comment:
nice, though I expected this to link to a wrapper component... that maps element ui props to our own props... that way if they define size `mini` we can translate that to our size `2xsmall` for example.. also won't break how it's used by other element ui elements. though button might be too simple for this..